testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/jms/JMSBridgeTest.java

```diff
@@ -136,6 +148,15 @@ public void sendAndReceiveMessage() throws Exception {
                 connection.close();
             }
         }
+    }
 
+    private long readMetrics() throws IOException {
```

Comment:
Can you also please add a smoke for second metrics attribute? (aborted message count)